### PR TITLE
Modified hw3/model_wrapper.py fix pytorch 0.5 version compatibility

### PR DESCRIPTION
**Pull request by TaegyunKwon**
In pytorch>=0.5, tensor.data[0] with zero dimensional tensor gives wrong value.
Fix: tensor.data[0] -> tensor.item()

UserWarning: invalid index of a 0-dim tensor. This will be an error in PyTorch 0.5. Use tensor.item() to convert a 0-dim tensor to a Python number

### CODE REVIEW

**Comment by TaegyunKwon** [comment]
Actually there were same issues in previous homeworks, but was not critical. However, it gives loss = 0 in HW3
